tests/common.py

```diff
@@ -533,31 +519,6 @@ def check_split_ixns(
         np.testing.assert_allclose(expected_u, sum_u_intra, rtol=rtol, atol=atol)
         np.testing.assert_allclose(expected_grad, sum_grad_intra, rtol=rtol, atol=atol)
 
-        # Compute the grads, potential with the ligand-water terms scaled
-        sum_grad_solv, sum_u_solv = compute_new_grad_u(
-            ffs.solv, precision, coords0, box, lamb, num_water_atoms, host_bps
-        )
-        WL_grad_solv, WL_u_solv = compute_ixn_grad_u(
-            ffs.solv,
-            precision,
-            coords0,
-            box,
-            lamb,
-            num_water_atoms,
-            host_bps,
-            water_idxs,
-            ligand_idxs,
-            protein_idxs,
-            is_solvent=True,
-        )
-
-        # U_solv = U_sum_ref - WL_ref + WL_solv
-        expected_u = sum_u_ref - WL_u_ref + WL_u_solv
-        expected_grad = sum_grad_ref - WL_grad_ref + WL_grad_solv
-
-        np.testing.assert_allclose(expected_u, sum_u_solv, rtol=rtol, atol=atol)
-        np.testing.assert_allclose(expected_grad, sum_grad_solv, rtol=rtol, atol=atol)
-
         # Compute the grads, potential with the protein-ligand terms scaled
```

Comment:
non-obvious loss of test functionality -- should this test be retained, but just replace `ffs.solv` with `ffs.prot` ?

(to assert that `ffs.prot` now controls both the WL and PL interaction group?)

Reply:
Updated the test to make it more clear what's happening here. Now that we only have L-E scaled together, the first part of the test (deleted) is doing the same check as the second, so it's not needed.